client/state/reader/posts/actions.js

```diff
@@ -24,13 +25,25 @@ function trackRailcarRender( post ) {
 }
 
 function fetchForKey( postKey ) {
+	const query = {};
+
+	const contentWidth = readerContentWidth();
+	if ( contentWidth ) {
+		query.content_width = contentWidth;
+	}
+
 	if ( postKey.blogId ) {
-		return wpcom.undocumented().readSitePost( {
-			site: postKey.blogId,
-			postId: postKey.postId,
-		} );
+		return wpcom.req.get( `/read/sites/${ postKey.blogId }/posts/${ postKey.postId }`, query );
 	}
-	return wpcom.undocumented().readFeedPost( postKey );
+	const { postId, feedId, ...params } = postKey;
+	return wpcom.req.get(
+		`/read/feed/${ encodeURIComponent( feedId ) }/posts/${ encodeURIComponent( postId ) }`,
```

Comment:
I think practically they are always numbers, but we encode here because historically we used to fetch feeds by URLs, too. I prefer keeping it just in case we still have an edge case that uses strings.